pyproject.toml

```diff
@@ -220,7 +216,7 @@ lint.select = [  # Base linting rule selections.
     "RUF001", "RUF002", "RUF003",  # Ambiguous unicode characters
     "RUF005",  # Encourages unpacking rather than concatenation
     "RUF008",  # Do not use mutable default values for dataclass attributes
-    "RUF011",  # Don't use static keys in dict comprehensions.
+    "B035",   # Don't use static keys in dict comprehensions.
```

Comment:
There is `"B0",  # Common mistakes and typos.` above, maybe you don't need it at all ?

Reply:
Good point. Removed!